ktor-network/jvm/src/io/ktor/network/sockets/SocketOptions.kt

```diff
@@ -117,6 +117,18 @@ sealed class SocketOptions(
      */
     class UDPSocketOptions internal constructor(customOptions: MutableMap<Any, Any?>) :
         PeerSocketOptions(customOptions) {
+        /**
+         * SO_BROADCAST socket option
+         */
+        var broadcast: Boolean = false
```

Comment:
are all udp kinds sockets can have this option set?

Reply:
Yes, SO_BROADCAST is strictly a datagram socket option.  You cannot send a message to the broadcast address without enabling it first.

Reply:
What about "bound" and "connected" udps?

Reply:
@cy6erGn0m what exactly would you like to know? I left a comment on this PR with my use case: https://github.com/ktorio/ktor/pull/1258#issuecomment-689026286.